tests/unit/community_label_propagation.c

```diff
@@ -32,7 +32,26 @@ int main() {
     igraph_vector_t weights;
     igraph_vector_int_t initial;
     igraph_vector_bool_t fixed;
-    igraph_integer_t i;
+    igraph_integer_t i, j;
+
+    /* Simple triangle graph, the output should be always one community */
+    igraph_small(&g, 0, IGRAPH_UNDIRECTED, 0,  1,  0,  2,  1,  2, -1);
+
+    for (j = 0; j < 100; j++) {
+        /* label propagation is a stochastic method */
+        igraph_rng_seed(igraph_rng_default(), j);
+
+        igraph_vector_int_init(&membership, 0);
```

Comment:
You now initialize the vector `membership` repeatedly in the for loop. A vector needs to be initialised only once. Could you please move this to outside the for loop (i.e. to L39)? Note that in L79 it now is also initialised still, that initialisation should be removed.